groups/bace-iot-platform/index.yml

```diff
@@ -0,0 +1,4 @@
+name: BACE IoT platform
```

Comment:
Because it is a product title I would write 'BACE IoT Platform' with the last word capitalized as well.